app/boards/intel_adsp_ace20_lnl.conf

```diff
@@ -0,0 +1,71 @@
+CONFIG_LUNARLAKE=y
+CONFIG_IPC_MAJOR_4=y
+
+# turn off SOF drivers
+CONFIG_INTEL_DMIC=n
+CONFIG_INTEL_DMIC_NHLT=n
+CONFIG_INTEL_SSP=n
+CONFIG_INTEL_ALH=n
+CONFIG_COMP_SRC=y
+
+CONFIG_COMP_SRC_IPC4_FULL_MATRIX=y
+
+# power settings
+CONFIG_PM=y
+CONFIG_PM_DEVICE=y
+CONFIG_PM_DEVICE_RUNTIME=y
+CONFIG_PM_DEVICE_POWER_DOMAIN=y
+CONFIG_PM_POLICY_CUSTOM=y
+
+CONFIG_POWER_DOMAIN=y
+CONFIG_POWER_DOMAIN_INTEL_ADSP=y
+
+# enable Zephyr drivers
+CONFIG_ZEPHYR_NATIVE_DRIVERS=y
+CONFIG_DAI=y
+CONFIG_DAI_INIT_PRIORITY=70
+CONFIG_DAI_INTEL_DMIC=y
+CONFIG_DAI_DMIC_HW_IOCLK=19200000
+CONFIG_DAI_INTEL_DMIC_NHLT=y
+CONFIG_DAI_DMIC_HAS_OWNERSHIP=n
+CONFIG_DAI_DMIC_HAS_MULTIPLE_LINE_SYNC=y
+CONFIG_DAI_INTEL_SSP=y
+CONFIG_DMA=y
+CONFIG_DMA_INTEL_ADSP_GPDMA=n
+CONFIG_INTEL_MODULES=y
+CONFIG_LIBRARY_MANAGER=y
+CONFIG_INTEL_ADSP_TIMER=y
+CONFIG_MM_DRV_INTEL_ADSP_TLB_REMAP_UNUSED_RAM=y
+
+CONFIG_HEAP_MEM_POOL_SIZE=8192
+CONFIG_L3_HEAP=y
+CONFIG_RIMAGE_SIGNING_SCHEMA="lnl"
+
+CONFIG_FORMAT_CONVERT_HIFI3=n
+CONFIG_PCM_CONVERTER_FORMAT_S16_C16_AND_S16_C32=y
+CONFIG_PCM_CONVERTER_FORMAT_S16_C32_AND_S32_C32=y
+CONFIG_PCM_CONVERTER_FORMAT_S16_C32_AND_S24_C32=y
+CONFIG_PCM_CONVERTER_FORMAT_S24_C24_AND_S24_C32=y
+CONFIG_PCM_CONVERTER_FORMAT_S24_C32_AND_S24_C24=y
+CONFIG_PCM_CONVERTER_FORMAT_S16_C32_AND_S16_C32=y
+CONFIG_LOG=y
+CONFIG_LOG_MODE_DEFERRED=y
+CONFIG_LOG_FUNC_NAME_PREFIX_INF=y
+CONFIG_COMP_VOLUME_WINDOWS_FADE=y
+CONFIG_COMP_UP_DOWN_MIXER=y
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=12000
```

Comment:
not sure what this is?

Reply:
I don't know how we can generate a 12kHz clock with the existing hardware?

Reply:
@plbossart This is ok, this is just used for Zephyr internal bookkeeping.